modules/post/osx/gather/enum_keychain.rb

```diff
@@ -21,7 +21,12 @@ def initialize(info = {})
         'License' => MSF_LICENSE,
         'Author' => [ 'ipwnstuff <e[at]ipwnstuff.com>', 'joev' ],
         'Platform' => [ 'osx' ],
-        'SessionTypes' => [ 'meterpreter', 'shell' ]
+        'SessionTypes' => [ 'meterpreter', 'shell' ],
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'SideEffects' => [ARTIFACTS_ON_DISK],
```

Comment:
> the user may see an authentication
          alert flash briefly on their screen that gets dismissed by a programmatically triggered click.

I believe we may also need `SCREEN_EFFECTS` here:
```suggestion
          'SideEffects' => [ARTIFACTS_ON_DISK, SCREEN_EFFECTS],
```

Reply:
Added